karma.conf.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./config/karma.config.js');
```

Comment:
What is this file good for? See Karma's docs:

> Unless provided as argument, the Karma CLI will look for a configuration file at
>
>    ./karma.conf.js
>    ./karma.conf.coffee
>    ./.config/karma.conf.js
>    ./.config/karma.conf.coffee
>
> in that order.

Isn't it easier to remove file `karma.conf.js` and rename `./config/karma.config.js` to `./config/karma.conf.js`?